locales/el/json.json

```diff
@@ -35,32 +35,32 @@
     "Address": "Διεύθυνση",
     "Address Line 2": "Γραμμή διεύθυνσης 2",
     "Administrator": "Διαχειριστής",
-    "Administrator users can perform any action.": "Οι χρήστες του διαχειριστή μπορούν να εκτελέσουν οποιαδήποτε ενέργεια.",
+    "Administrator users can perform any action.": "Οι διαχειριστές μπορούν να εκτελέσουν οποιαδήποτε ενέργεια.",
     "Afghanistan": "Αφγανιστάν",
     "Aland Islands": "Νήσοι Ώλαντ",
     "Albania": "Αλβανία",
     "Algeria": "Αλγερία",
-    "All of the people that are part of this team.": "Όλοι οι άνθρωποι που είναι μέρος αυτής της ομάδας.",
+    "All of the people that are part of this team.": "Όλοι όσοι είναι μέρος αυτής της ομάδας.",
     "All resources loaded.": "Όλοι οι πόροι φορτώθηκαν.",
     "All rights reserved.": "Πνευματική προστασία περιεχομένου.",
     "Already registered?": "Είστε ήδη εγγεγραμμένος?",
     "American Samoa": "Αμερικανική Σαμόα",
     "An error occurred while uploading the file.": "Παρουσιάστηκε σφάλμα κατά τη μεταφόρτωση του αρχείου.",
     "An error occurred while uploading the file: :error": "Παρουσιάστηκε σφάλμα κατά τη μεταφόρτωση του αρχείου: :error",
     "An unexpected error occurred and we have notified our support team. Please try again later.": "Παρουσιάστηκε απροσδόκητο σφάλμα και έχουμε ειδοποιήσει την ομάδα υποστήριξής μας. Παρακαλώ προσπάθησε ξανά αργότερα.",
-    "Andorra": "Ανδόρας",
+    "Andorra": "Ανδόρρα",
     "Angola": "Αγκόλα",
     "Anguilla": "Ανγκουίλλα",
     "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "Ένας άλλος χρήστης έχει ενημερώσει αυτόν τον πόρο από τότε που φορτώθηκε αυτή η σελίδα. Παρακαλώ ανανεώστε τη σελίδα και προσπαθήστε ξανά.",
     "Antarctica": "Ανταρκτική",
     "Antigua And Barbuda": "Αντίγκουα και Μπαρμπούντα",
     "Antigua and Barbuda": "Αντίγκουα και Μπαρμπούντα",
-    "API Token": "Διακριτικό API",
-    "API Token Permissions": "Άδειες συμβόλων API",
-    "API Tokens": "Μάρκες API",
+    "API Token": "API Token",
```

Comment:
Add this value to the "_excludes.json" file, otherwise the translation status will show it.

The file is located in the localization folder.